launch_testing/launch_testing/__init__.py

```diff
@@ -0,0 +1,61 @@
+import io
+import re
+from launch.output_handler import LineOutput
+
+
+class InMemoryHandler(LineOutput):
+
+    PREFIXES_TO_FILTER = [
```

Comment:
These prefixes should be able to be adjusted or at least disabled. I'd suggest a constructor argument with these default values. And call these default values something like DDS_PREFIXES or some other semantically correct name.